app/models/interview.rb

```diff
@@ -20,7 +20,12 @@ class Interview < ApplicationRecord
   after_update :resend_confirmation
 
   default_scope { order :scheduled }
-  scope :pending, -> { where completed: false }
+  scope :pending, lambda {
+                    joins(:application_submission).where(interviews: { completed: false },
+                                                         application_submission: { saved_for_later: false })
+                  }
+
+  delegate :saved_for_later?, to: :application_submission
```

Comment:
Do you ever actually call `@interview.saved_for_later?` anywhere? Like why have this delegation, I wonder?

Reply:
Must have forgotten to take it out, will remove